ansible/roles/ha-ip-update/tasks/main.yml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2021 The SODA Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+- name: Check that the gelato-ha.yml exists
+  stat:
+    path: "{{ role_path }}/../../group_vars/gelato-ha.yml"
+  register: gelatoha_stat
+
+
+- name: replace the api_ip and s3_ip in the gelato-ha.yml
+  hosts: controllers
+  replace:
+    path: "{{ role_path }}/../../group_vars/gelato-ha.yml"
+    regexp: "{{ item.regexp }}"
+    replace: "{{ item.replace }}"
+  with_items:
+    - { regexp: 'API_IP', replace: '{{ hostvars["k8smaster"]["soda_api_ip"] }}' }
```

Comment:
I see how this is working now. As per previous comments please rename the env variable to indicate it is multicloud HA  API and S3 IP and port.

Reply:
Done